### PR TITLE
Pr113x L1T Bug fix:  uGT emulator of displaced muons

### PR DESCRIPTION
**Pull request by rekovic**
#### PR description:
PR 11_3_X 
This is a bug fix needed for MWGR: first check if the high value of the unconstrained pT cut-window is valid (i.e. greater than 0), before proceeding to evaluate the cut on unconstrained pT, otherwise the emulation of triggers without unconstrained pT cuts specified will be affected.
There are no changes to output.
This PR does not depend on any externals.

#### if this PR is a backport please specify the original PR and why you need to backport that PR:

Before submitting your pull requests, make sure you followed this checklist:
- verify that the PR is really intended for the chosen branch
- verify that changes follow [CMS Naming, Coding, And Style Rules](http://cms-sw.github.io/cms_coding_rules.html)
- verify that the PR passes the basic test procedure suggested in the [CMSSW PR instructions](https://cms-sw.github.io/PRWorkflow.html)

### CODE REVIEW

**Comment by rekovic** [comment]
please test

**Comment by silviodonato** [comment]
urgent
needed for MWGR

File: L1Trigger/L1TGlobal/src/MuCondition.cc [hunk 1]
```diff
@@ -382,12 +382,28 @@ const bool l1t::MuCondition::checkObjectParameter(const int iCondition,
...
-    LogDebug("L1TGlobal") << "\t\t Muon Failed unconstrainedPt checkThreshold " << std::endl;
-    return false;
+  if (objPar.unconstrainedPtHigh > 0)  // Check if unconstrained pT cut-window is valid
+  {
+    if (!checkThreshold(objPar.unconstrainedPtLow,
+                        objPar.unconstrainedPtHigh,
+                        cand.hwPtUnconstrained(),
+                        m_gtMuonTemplate->condGEq())) {
+      LogDebug("L1TGlobal") << "\t\t Muon Failed unconstrainedPt checkThreshold " << std::endl;
+      std::cout << "\t\t Muon Failed unconstrainedPt checkThreshold; iCondition = " << iCondition << std::endl;
```
Comment (davidlange6) [inline L1Trigger/L1TGlobal/src/MuCondition.cc:392]: is the cout needed?

Comment (rekovic) [reply L1Trigger/L1TGlobal/src/MuCondition.cc:392]: nope.

Comment (rekovic) [reply L1Trigger/L1TGlobal/src/MuCondition.cc:392]: updating

**Comment by rekovic** [comment]
please test

**Comment by rekovic** [comment]
will make backport to 11_2_X as soon as IB is available.

**Comment by rekovic** [comment]
+1

**Comment by qliphy** [comment]
+1

**Pull request merged**